src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/TestFixture.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.IO;
+using Common.Logging;
+using Common.Logging.Serilog;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json.Linq;
+using Serilog;
+using Xunit;
+using Assert = Microsoft.VisualStudio.TestTools.UnitTesting.Assert;
+
+namespace Catalyst.Core.Modules.Dfs.Tests.IntegrationTests
+{
+    public class TestFixture
+    {
+        private const string passphrase = "this is not a secure pass phrase";
+        public static IpfsEngine Ipfs = new IpfsEngine(passphrase.ToCharArray());
+        public static IpfsEngine IpfsOther = new IpfsEngine(passphrase.ToCharArray());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4612924)